src/pages/resultsView/ResultsViewPageHelpers.ts

```diff
@@ -102,8 +102,11 @@ export function populateSampleSpecificationsFromVirtualStudies(samplesSpecificat
         }) as SamplesSpecificationElement[][];
     }));
 
-    // ts not resolving type above and not sure why, so cast it
-    samplesSpecification = samplesSpecification.concat(allVirtualStudySampleSpecs as SamplesSpecificationElement[]);
+    // concatenate allVirtualStudySampleSpecs only when we don't have any samplesSpecifications
+    if (samplesSpecification.length === 0) {
```

Comment:
if we have a mixture of virtual studies and physical studies, there might be a problem b/c there will be sampleSpecifications for the physical studies.

Reply:
please try that possibility and make sure we handle it properly